File: Data Structures & Algorithms/two-integer-sum/submission-12.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {

        //sort(nums.begin(), nums.end()); //sorts numbers in nums from lowest to highest

        for(int i = 0; i < nums.size(); i++){

            int value = target - nums[i];

            for(int j = i + 1; j< nums.size(); j++){
                if(value == nums[j]){
                    return {i, j};
                }
            }

        }
        return {};










        
    }
};

File: Data Structures & Algorithms/two-integer-sum-ii/submission-12.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:

        left = 0
        right = len(numbers) - 1

        while left < right:

            while left < right:
                #value = target - numbers[right]
                if numbers[right] + numbers[left] == target:
                    left += 1
                    right += 1
                    return [left, right]
                right -= 1

            left += 1
            right = len(numbers) - 1



        
        else:
            return []









        


        




































    
        # left = 0
        # right = len(numbers) - 1

        # while left < right:
        
        #     while left < right:
        #         total = numbers[left] + numbers[right]

        #         if total == target:
        #             return [left+ 1, right+ 1]
                
        #         right -= 1

        #     left +=1
        #     right = len(numbers) -1






        